tests/lib/src/common_utils.cpp

```diff
@@ -5,54 +5,106 @@
 #include "common_utils.h"
 
 #include <inference_engine.hpp>
+#include <openvino/runtime/infer_request.hpp>
```

Comment:
maybe it s better to add #include <openvino/openvino.hpp> in header file only?

Reply:
Fixed